staging/src/k8s.io/pod-security-admission/test/fixtures_test.go

```diff
@@ -61,11 +61,23 @@ func TestFixtures(t *testing.T) {
 			failDir := filepath.Join("testdata", string(level), fmt.Sprintf("v1.%d", version), "fail")
 
 			// render the minimal valid pod fixture
-			validPod, err := GetMinimalValidPod(level, api.MajorMinorVersion(1, version))
+			osNeutralPod, err := GetMinimalValidOSNeutralPod(level, api.MajorMinorVersion(1, version))
 			if err != nil {
 				t.Fatal(err)
 			}
-			expectedFiles.Insert(testFixtureFile(t, passDir, "base", validPod))
+			expectedFiles.Insert(testFixtureFile(t, passDir, "os-neutral-base", osNeutralPod))
+
+			linuxPod, err := GetMinimalValidLinuxPod(level, api.MajorMinorVersion(1, version))
+			if err != nil {
+				t.Fatal(err)
+			}
+			expectedFiles.Insert(testFixtureFile(t, passDir, "linux-base", linuxPod))
```

Comment:
name `base_linux.yaml` and `base_windows.yaml` to sort by the neutral base.yaml file